parachain/pallets/omni-account/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020-2024 Trust Computing GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use core_primitives::Identity;
+use parity_scale_codec::{Decode, Encode};
+
+sp_api::decl_runtime_apis! {
+	pub trait OmniAccountApi<AccountId>
+	where
+		AccountId: Encode + Decode,
```

Comment:
NIT: we could do `use parity_scale_codec::Codec;`

```suggestion
		AccountId: Codec,
```